dialog.cpp

```diff
@@ -87,6 +114,9 @@ void Dialog::checkSpelling()
                 case SpellCheckDialog::ReplaceOnce:
                     cursor.insertText(checkDialog->replacement());
                     break;
+                case SpellCheckDialog::ReplaceAll:
+                    this->replaceAll(cursor.position(), word, checkDialog->replacement());
```

Comment:
I think we should remove the this-> part (just to be consistent with the rest of the code.)